.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,20 @@
 Hello & Welcome to our repo,
 
-You Can find a link to our review process [here](https://github.com/Azure/adx-documentation-pr/wiki/swagger-management). if you can't access this link, register your github your account to [Azure Github](https://aka.ms/azuregithub).
+If you're a Microsoft Employee, please review and ensure that the following steps have been taken:
+
+- Is this a PR for a new management (RP or New API Version)? (If your answer is Yes then if you haven't done so already contant the ARM team for review).
+- If this a new Data-Plane Swagger, have you gone through an review from the API review board? (If you're answer is no then contant the API review board to have it reviwed)
```

Comment:
typo: an review -> a review
typo: contant -> contact
should there be a link here on how to contact the API review board?